cli/package.json

```diff
@@ -5,6 +5,9 @@
   "scripts": {
     "clean": "make clean",
     "build": "make",
+    "build:go": "make go-turbo",
+    "build:rust": "cargo build -p turbo",
+    "build:go-copy": "echo $PWD && ls ../target/debug && mv go-turbo ../target/debug/go-turbo",
```

Comment:
We should not merge this as is. It does not include Windows handling or debug/release profile handling